doc/ReleaseNotes/ReleaseNotes.tex

```diff
@@ -192,17 +192,17 @@ \section{Changes Introduced in this Release}
 	
 	\underline{NEW FUNCTIONALITY}
 	\begin{itemize}
-		\item xxx
+		\item A new Viscosity (VSC) package for the Groundwater Flow (GWF) Model is introduced in this release.  The effects of viscosity are accounted for by updates intercell conductance, as well as the conductance between the aquifer and head-dependent boundaries, based on simulated concentrations and\/or temperatures.  As with other packages, the viscosity package is activated by specifying ``VSC6'' as the file type in a GWF name file.  Testing of the VSC Package is accomplished using example test problems; however, changes to the code and input may be required in response to user needs and testing.
 	%	\item xxx
 	%	\item xxx
 	\end{itemize}
 
-	%\underline{EXAMPLES}
-	%\begin{itemize}
-	%	\item xxx
+	\underline{EXAMPLES}
+	\begin{itemize}
+		\item Four new example problems added to the ``autotest'' folder that tests the functionality of the new Viscosity (VSC) Package and named test\_gwf\_vsc01, test\_gwf\_vsc02, test\_gwf\_vsc03\_sfr, and test\_gwf\_vsc04\_lak
```

Comment:
Autotests are not the examples.  Autotests are not part of the distribution.  This section is for new examples that would be in the modflow6-examples repository.

Reply:
Addressed in 835699c